x-pack/docs/en/security/authorization/mapping-roles.asciidoc

```diff
@@ -28,6 +28,8 @@ you are able to map users to both API-managed roles and file-managed roles
 NOTE: The PKI, LDAP, Kerberos and SAML realms support using
 <<authorization_realms, authorization realms>> as an alternative to role mapping.
 
+NOTE: Users with no roles assigned will be unauthorized for any action.
```

Comment:
True, but... if anonymous is enabled then all users have the anonymous role, so this sentence might confuse someone.
Perhaps, we need another NOTE (though I worry about overwhelming people with so many notes, so maybe just a regular sentence) that if anonymous is enabled, all users receive that role in addition to their mapped roles.

Reply:
Nice catch! I did add another note:
```
NOTE: When {ref}/anonymous-access.html[anonymous access] is enabled the roles
of the anonymous user are mapped to all the other users as well.
```
This is important IMO.